### PR TITLE
Activate cypress workflow step

### PR DESCRIPTION
**Pull request by daniel-va**

### CODE REVIEW

File: .github/workflows/code-quality.yml [hunk 1]
```diff
@@ -146,38 +77,19 @@ jobs:
...
         run: |
           cd development
+          chmod +x ./init/elasticsearch/init.sh
+          sed -i 's/- \.\/volumes\/elasticsearch\/data:\/usr\/share\/elasticsearch\/data//g' ./docker-compose.yaml
           docker compose up -d db oidc elasticsearch
-      - name: Setup
-        run: |
-          npm run prisma -- reset -f
-          # npm run start:server &> /dev/null &
+          sleep 120
```
Comment (vej-ananas) [inline .github/workflows/code-quality.yml:89]: This necessary?

File: README.md [hunk 2]
```diff
@@ -13,44 +13,31 @@ The following components must be installed on the development computer:
 Follow these steps to set up the development environment on your local machine:
 
 - [1. Configure Local Systems](#1-Configure-Local-Systems)
-- [2. Configure the Asset Server](#2-Configure-the-Asset-Server)
-- [3. Install Dependencies](#3-Install-Dependencies)
-- [4. Generate Database Types](#4-Generate-Database-Types)
-- [5. Initialize MinIO](#5-Initialize-MinIO)
+- [3. Install Dependencies](#2-Install-Dependencies)
```
Comment (vej-ananas) [inline README.md:16]: Wrong numbers

File: e2e/cypress/e2e/common/give-admin-perm.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { Given } from '@badeball/cypress-cucumber-preprocessor';
+import { assetsBaseUrl } from '../../support/config/config';
+
+Given('User has admin permissions', () => {
+  const command = `docker exec -i swissgeol-assets-postgres psql -U postgres -d postgres -a -f ./give_admin_perm.sql`;
+  cy.exec(command).then((result) => {
+    console.log(result);
```
Comment (vej-ananas) [inline e2e/cypress/e2e/common/give-admin-perm.ts:7]: Is `console.log` desired here?

File: e2e/cypress/support/commands/helper.commands.ts [hunk 4]
```diff
@@ -14,36 +16,24 @@ import { assetsBaseUrl } from '../config/config';
...
+      cy.contains('button', 'Login').click({ force: true });
       cy.wait('@token')
         // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
         .then((interception) => interception.response.body.access_token)
         // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
         .then((token) => window.localStorage.setItem('access_token', token));
     },
     {
-      validate() {
+      /*      validate() {
```
Comment (vej-ananas) [inline e2e/cypress/support/commands/helper.commands.ts:36]: Do not comment out code - remove or fix

**Review by vej-ananas — CHANGES_REQUESTED** [review]

File: .github/workflows/code-quality.yml (hunk 1 above)
Comment (DmByK) [reply .github/workflows/code-quality.yml:89]: I have reduced but is still needed for services to be up and running before Cypres starts.

**Review by vej-ananas — APPROVED** [review]

**Pull request merged**